src/SFA.DAS.EmployerAccounts/Models/Account/LegalEntity.cs

```diff
@@ -9,9 +9,9 @@ public class LegalEntity
         public virtual long Id { get; set; }
         public virtual string Code { get; set; }
         public virtual DateTime? DateOfIncorporation { get; set; }
-        public virtual string Name { get; set; }
+        //public virtual string Name { get; set; }
```

Comment:
i was getting exception ,  'Name' and 'RegisteredAddress' not used . After removing i ran tests all looks  good. hence removed to avoid the confusion between AccountLegalEntity name and 
LegalEntityName(no such thing exist)